client/blocks/term-form-dialog/index.jsx

```diff
@@ -134,11 +135,42 @@ class TermFormDialog extends Component {
 		this.props.bumpStat( 'taxonomy_manager', statLabels.mc );
 		this.props.recordGoogleEvent( 'Taxonomy Manager', statLabels.ga );
 
-		savePromise.then( ( savedTerm ) => {
-			this.setState( { saving: false } );
-			this.props.onSuccess( savedTerm );
-			this.closeDialog();
-		} );
+		savePromise
+			.then( ( savedTerm ) => {
+				this.setState( { saving: false } );
+				this.props.onSuccess( savedTerm );
+				this.closeDialog();
+
+				const successText = isNew
+					? translate( 'Added new %(taxonomyName)s successfully.', {
+							args: { taxonomyName: taxonomy },
+					  } )
+					: translate( 'Updated %(taxonomyName)s successfully.', {
+							args: { taxonomyName: taxonomy },
+					  } );
+
+				this.props.successNotice( successText, {
+					id: 'taxonomy-manager-save',
+					duration: 4000,
+				} );
```

Comment:
Sure! I only added it because I felt maybe it was better for an errorNotice to be accompanied by a successNotice, but your reasoning about the changes being visible makes total sense. :)

Reply:
Thank you!